modules/oe_content_publication/oe_content_publication.post_update.php

```diff
@@ -29,3 +32,71 @@ function oe_content_publication_post_update_00001_update_field_labels(array &$sa
 function oe_content_publication_post_update_00002() {
   \Drupal::service('module_installer')->install(['oe_content_documents_field']);
 }
+
+/**
+ * Create the new fields in the Publication content type.
+ */
+function oe_content_publication_post_update_00003(): void {
+  $storage = new FileStorage(drupal_get_path('module', 'oe_content_publication') . '/config/post_updates/00003_create_fields');
+  \Drupal::service('config.installer')->installOptionalConfig($storage);
+}
+
+/**
+ * Update field labels.
+ */
+function oe_content_publication_post_update_00004(): void {
+  $new_field_labels = [
+    'node.oe_publication.oe_publication_type' => 'Resource type',
+    'node.oe_publication.oe_documents' => 'Files',
+  ];
+  foreach ($new_field_labels as $id => $new_label) {
+    $field_config = FieldConfig::load($id);
+    $field_config->setLabel($new_label);
+    $field_config->save();
+  }
+
+  $new_translatable_settings = [
+    'node.oe_publication.oe_summary' => FALSE,
+    'node.oe_publication.oe_teaser' => FALSE,
+    'node.oe_publication.oe_publication_type' => TRUE,
```

Comment:
Reference fields are always untranslatable. node.oe_publication.oe_publication_type can not be translatable.
Instead of it you should update `field.storage.node.oe_publication_type.yml` and set `translatable: false`.

Reply:
This is fixed.